pkgs/applications/audio/mod-distortion/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, lv2 }:
+
+stdenv.mkDerivation rec {
+  name = "mod-distortion-${version}";
+  version = "git-2015-05-18";
+
+  src = fetchFromGitHub {
+    owner = "portalmod";
+    repo = "mod-distortion";
+    rev = "0cdf186abc2a9275890b57057faf5c3f6d86d84a";
+    sha256 = "1wmxgpcdcy9m7j78yq85824if0wz49wv7mw13bj3sw2s87dcmw19";
+  };
+
+  buildInputs = [ lv2 ];
+
+  installPhase = ''
+    make install LV2_PATH=$out/lib/lv2
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Removing `homepage` will not lead to the expected result. The commit mentioned by @jagajaga allows you to write `inherit (src.meta) homepage` instead of repeating the url.